src/network/index.ts

```diff
@@ -0,0 +1,37 @@
+import {NetworkConfiguration} from "./interfaces";
+import {Configuration} from "../configuration";
+
+export const kusamaConfiguration: NetworkConfiguration = {
+  addressPrefix: 1,
+  unit: {
+    image: "https://svgur.com/i/Jiy.svg",
+    symbol: "KSM",
+  },
+  wsRpcUrl: "wss://kusama-rpc.polkadot.io/",
+};
+
+export const westendConfiguration: NetworkConfiguration = {
+  addressPrefix: 1,
```

Comment:
I don't think this is same as Kusama. Think Westend is 42 and Kusama 2

Reply:
https://wiki.polkadot.network/docs/en/build-protocol-info

Reply:
As I mentioned inside PR description

> Westend configuration is a placeholder for now (I will set real values on next task #43)

but If you want I can refactor this now?

Reply:
You can either add proper values now or remove westend configuration from this PR